azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -1011,6 +1020,76 @@ private void ajaxExecuteFlow(final HttpServletRequest req,
     ret.put("execid", exflow.getExecutionId());
   }
 
+  /**
+   * This method is used to update the property. propType: Is the umbrella for properties for which
+   * values need to be updated subType: Actual property for which values need to be updated val:
+   * value to be updated
+   * <p>
+   * Example: propType=containerDispatch&subType=updateAllowList&val=spark,java
+   */
+  private void ajaxUpdateProperty(final HttpServletRequest req,
+      final HttpServletResponse resp, final HashMap<String, Object> ret, final User user)
+      throws ServletException {
+    try {
+      if (!HttpRequestUtils.hasPermission(this.userManager, user, Type.ADMIN)) {
+        ret.put("error", String.format("User %s doesn't have ADMIN permission for updating "
+            + "property", user));
+      }
+      String propType = getParam(req, "propType");
+      if (propType.equals("containerDispatch")) {
+        updateContainerDispatchProps(req, ret);
+      } else {
+        ret.put("error", "Unsupported propType: " + propType);
+      }
+    } catch (final Exception e) {
+      e.printStackTrace();
+      ret.put("error", "Error on update property. " + e.getMessage());
+    }
+  }
+
+  private void updateContainerDispatchProps(final HttpServletRequest req,
+      final HashMap<String, Object> ret)
+      throws ServletException {
+    if (this.executorManagerAdapter instanceof ContainerizedDispatchManager) {
+      ContainerizedDispatchManager containerizedDispatchManager = (ContainerizedDispatchManager) this.executorManagerAdapter;
+      String subType = getParam(req, "subType");
+      String val = getParam(req, "val");
+      switch (subType) {
+        case "updateAllowList":
+          containerizedDispatchManager.updateAllowList(getSetFromString(val));
+          break;
+        case "appendAllowList":
+          containerizedDispatchManager.appendAllowList(getSetFromString(val));
+          break;
+        case "updateDenyList":
+          containerizedDispatchManager.updateDenyList(getSetFromString(val));
+          break;
+        case "appendDenyList":
+          containerizedDispatchManager.appendDenyList(getSetFromString(val));
+          break;
+        case "updateRampUp":
+          containerizedDispatchManager.setRampUp(Integer.parseInt(val));
+          break;
+        default:
+          ret.put("error", "Unsupported subType: " + subType);
+          break;
+      }
+    } else {
+      ret.put("error",
+          "ExecutorManagerAdapter is not of type: " + ContainerizedDispatchManager.class.getName());
```

Comment:
Better to check it before calling this method.

Reply:
done